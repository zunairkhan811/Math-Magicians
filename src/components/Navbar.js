import { NavLink } from 'react-router-dom';

const Navbar = () => (
  <>
    <nav>
      <div className="nav-logo">
        <h1>Math Magicians</h1>
      </div>
      <div className="nav-links">
        <ul>
          <li>
            <NavLink to="/Home" className="links">Home</NavLink>
          </li>
          <li>
            <NavLink to="/Calculator" className="links">Calculator</NavLink>
          </li>
          <li>
            <NavLink to="/Quote" className="links">Quote</NavLink>
          </li>
        </ul>
      </div>
    </nav>
  </>
);

export default Navbar;
