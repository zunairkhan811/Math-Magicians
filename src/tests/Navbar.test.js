import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

test('renders Navbar component correctly', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,

  );
  expect(asFragment()).toMatchSnapshot();
});
