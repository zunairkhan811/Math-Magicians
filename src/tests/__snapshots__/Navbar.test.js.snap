// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navbar component correctly 1`] = `
<DocumentFragment>
  <nav>
    <div
      class="nav-logo"
    >
      <h1>
        Math Magicians
      </h1>
    </div>
    <div
      class="nav-links"
    >
      <ul>
        <li>
          <a
            class="links"
            href="/Home"
          >
            Home
          </a>
        </li>
        <li>
          <a
            class="links"
            href="/Calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            class="links"
            href="/Quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </div>
  </nav>
</DocumentFragment>
`;
